pkg/cmd/admin.go

```diff
@@ -54,22 +64,40 @@ func newAdminDataCommand() *cobra.Command {
 
 			return nil
 		},
+		RunE: backup,
 	}
 	v := version.Get()
-	data.PersistentFlags().String("storage", config.BackupsDir, "Directory with backups")
-	data.PersistentFlags().String("name",
+	cmd.PersistentFlags().String("storage", config.BackupsDir, "Directory with backups")
+	cmd.PersistentFlags().String("name",
```

Comment:
Is it useful to have a default for the name of the thing being restored?

Reply:
These flags are for backup only, restore expects full path - not really sure if that's a good or bad UX... I'm not a fan.
Nice to have could be to provide a subcommand or opt (`--list`) that would output a list of backups in the `--storage`, or even have a completion for backup name.

To provide a default backup name to restore, we'd have to scan the `microshift-backups/` and pick a right one (last one?). I don't see a good default backup name for restoring when there's more than one backup

Reply:
OK, I thought part of the reason they were changing was because they were being reused but I see now that's just a variable name change. Using the full path as input argument makes the most sense. I agree, having a meaningful default would be hard.